templates/.github/workflows/releaser.yml

```diff
@@ -2,7 +2,6 @@ name: Releaser
 on:
   push:
     paths: [ 'version.json' ]
-    branches: [ $default-branch ]
```

Comment:
As far as I can tell, this should be safe to roll out: Once we merge this PR (to `master`), repos will start using the new `releaser` workflow, even if they haven't merged the PR updating the template yet. This means that `releaser` is only run for PRs merged into the default branch (which is the current behavior).